cmd/clusterctl/examples/openstack/machines.yaml.template

```diff
@@ -57,7 +66,15 @@ items:
           namespace: openstack-provider-system
         trunk: false
         tags:
-          - <Your Tag>
+          cluster:
+            tags:
+              - a_tag_applied_to_all_created_resources
+          server:
+            tags:
+              - a_tag_only_applied_to_servers
+          network:
+            tags:
+              - a_tag_only_applied_to_network_resources
```

Comment:
sorry but I don't understand why cluster/network tag applied in this machine template? my understanding cluster/network should only apply to cluster ?

Reply:
cluster, server, and network are seperate. Cluster tags get applied to all created resources. Server tags get applied to servers, and network tags only get applied to networks.

Reply:
understood, but this applied in machine.yaml.template makes me confusing, should cluster tags applied to cluster wide yaml or something like that?

Reply:
my new revision will address this